django/contrib/admin/checks.py

```diff
@@ -85,6 +86,36 @@ def check(self, admin_obj, **kwargs):
             *self._check_readonly_fields(admin_obj),
         ]
 
+    def _check_manager_name(self, obj):
+        if not isinstance(obj.manager_name, str):
```

Comment:
just a thought, but I feel like this might be slightly easier to follow if the `if not isinstance(obj.manager_name, str)` portion wiere  in the `else` portion of the `try/except/else` and the outer `if/else` were removed. Not a blocker, just an opinion :)